nextcloudappstore/core/templates/app/detail.html

```diff
@@ -93,11 +93,12 @@
             <section class="app-download">
                 <h4 id="downloads">{% trans "Downloads" %}</h4>
                 <table class="table">
-                    {% for platform_v, release in latest_releases_by_platform_v|sort_by_version:'desc' %}
-                        {% if release %}
+                    {% for platform_v, releases in latest_releases_by_platform_v|sort_by_version:'desc' %}
+                        {% if releases.stable or releases.nightly %}
                             <tr>
                                 <td>Nextcloud {{ platform_v }}</td>
-                                <td><a href="{{ release.download }}" rel="noopener noreferrer">Latest release ({{ release.version }})</a></td>
+                                <td>{% if releases.stable %}<a href="{{ releases.stable.download }}" rel="noopener noreferrer">Latest release ({{ releases.stable.version }})</a>{% endif %}</td>
```

Comment:
Maybe we should drop the preceding "Latest release" and "Nightly" and just use "x.x.x" and "x.x.x-nightly (unstable)" since it takes up a lot of screen space